### PR TITLE
Build the tree and modern VSIX together and a few other things

### PR DESCRIPTION
**Pull request by davkean**
- Remove unneeded attributes
- Log a few more things
- Build.cmd now builds the tree and modern VSIX together
- Add /build switch for "implicit" build
- Seperate config from build options			
- Don't -> Does not			
- Remove trailing ' from log output
- Nuget -> NuGet to match file on disk			
- Fix spacing in build.proj

### CODE REVIEW

**Comment by davkean** [comment]
tag @dotnet/project-system

**Comment by davkean** [comment]
@333fred Are MicroBuilds builds different between branches? If not, then I'll need to continue to support the old /BuildModernVsixPackages switch to continue to support master.

**Comment by 333fred** [comment]
The steps are the same between each branch. We can update it to not call `/BuildModernVsixPackage` if we update all the branches. Also, please retarget to master, dev15.1.x, or dev15.0.x.

**Comment by davkean** [comment]
@333fred  Do we actually have MicroBuilds running for the other branches at this point?

File: build.cmd [hunk 1]
```diff
@@ -46,36 +45,45 @@ if "%VisualStudioVersion%" == "" (
...
 set BinariesDirectory=%Root%bin\%BuildConfiguration%\
-set LogFile=%BinariesDirectory%Build.log
 if not exist "%BinariesDirectory%" mkdir "%BinariesDirectory%" || goto :BuildFailed
 
-msbuild /nologo /warnaserror /nodeReuse:%NodeReuse% /consoleloggerparameters:Verbosity=minimal /fileLogger /fileloggerparameters:LogFile="%LogFile%";verbosity=diagnostic /t:"%MSBuildTarget%" /p:Configuration="%BuildConfiguration%" /p:RunTests="%RunTests%" /p:DeployVsixExtension="%DeployVsixExtension%" "%Root%build\build.proj" %MSBuildAdditionalArguments%
-if ERRORLEVEL 1 (
+REM We need to build the VSIX and Modern VSIX packages in a different MSBuild process because
+REM MicroBuild when building the modern VSIX packages has a dependency on a dll with the same 
+REM version but different contents.
+for %%T IN (%MSBuildTarget%, BuildModernVsixPackages) do (
```
Comment (basoundr) [inline build.cmd]: `target` instead of just a `T`?

**Comment by 333fred** [comment]
@davkean yeah, we have it enabled for `master`, `dev15.0.x`, and `dev15.1.x`.

File: build.cmd (hunk 1 above)
Comment (davkean) [inline build.cmd]: The batch language requires this variable to be a single character.

**Comment by davkean** [comment]
Closed in lieu of https://github.com/dotnet/roslyn-project-system/pull/1684.

**Comment by basoundr** [comment]
@davkean Guess you need to close this?

**Pull request closed without merging**